test/functional/fixtures/regression/gh-1054/test.js

```diff
@@ -4,8 +4,9 @@ describe('[Regression](GH-1054)', function () {
             return runTests('testcafe-fixtures/index.test.js', 'Type text in the input');
         });
 
+        // NOTE: IE11 does not raise Input event on contenteditable element
         it('Typing in the content editable element with "replace" option', function () {
-            return runTests('testcafe-fixtures/index.test.js', 'Type text in the content editable element');
+            return runTests('testcafe-fixtures/index.test.js', 'Type text in the content editable element', { skip: [ 'ie' ] });
```

Comment:
I don't get it. I can paste text in contenteditable by `ctrl+v` shortcut in IE. I think we should execute `typeText` action with `replace` option for IE. Do we?

Reply:
this test checks if input event happened. But in IE in contenteditable element input event never raised. So test is incorrect now for ie. We can dublicate this test for IE only without with additional check

Reply:
yes